### PR TITLE
[dom-gpu] Adding AMBER CUDA pat version

### PR DESCRIPTION
**Pull request by victorusu**
This build should be done in the compute node. It does not work in the login nodes.

### CODE REVIEW

File: easybuild/easyconfigs/a/Amber/Amber-16-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ('cray-netcdf/4.4.1.1.3', EXTERNAL_MODULE),
+    ('flex', '2.6.4'),
+    ('perftools-lite-gpu', EXTERNAL_MODULE),
+]
+
+buildininstalldir = True
+
+prebuildopts = 'cd .. && mv amber%s/* . && ' % version
+prebuildopts += 'export AMBERHOME=%(builddir)s; export CUDA_HOME=$CUDATOOLKIT_HOME;'
+prebuildopts += './configure -mpi -cuda -noX11 -crayxt5 --skip-python --with-netcdf $EBROOTNETCDF gnu <<< Y;'
```
Comment (jgphpc) [inline easybuild/easyconfigs/a/Amber/Amber-16-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb]: What means this ? <<<Y

Comment (victorusu) [reply easybuild/easyconfigs/a/Amber/Amber-16-CrayGNU-17.08-cuda-8.0-pat-6.5.1.eb]: It means, yes to the question it asks in the command line.
The question is "do you want to apply the update patches in Amber?"
And we say yes.

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
retest this please

**Comment by victorusu** [comment]
[This has passed already!](https://jenkins.cscs.ch/view/ProductionEB/job/TestingEB/563/label=dom/console)  Interesting...

**Pull request merged**